tests/unit/MigrationTest.js

```diff
@@ -464,6 +464,56 @@ describe('Migrations', () => {
                         },
                     });
                 }));
+
+        it('Should remove any instances of lastActorEmail found in a report', () =>
+            Onyx.multiSet({
+                [`${ONYXKEYS.COLLECTION.REPORT}1`]: {
+                    reportID: 1,
+                    lastActorEmail: 'fake@test.com',
+                    lastActorAccountID: 5,
+                },
+            })
+                .then(PersonalDetailsByAccountID)
+                .then(() => {
+                    expect(LogSpy).toHaveBeenCalledWith('[Migrate Onyx] PersonalDetailsByAccountID migration: removing lastActorEmail from report 1');
+                    const connectionID = Onyx.connect({
+                        key: ONYXKEYS.COLLECTION.REPORT,
+                        waitForCollectionCallback: true,
+                        callback: (allReports) => {
+                            Onyx.disconnect(connectionID);
+                            const expectedReport = {
+                                reportID: 1,
+                                lastActorAccountID: 5,
+                            };
+                            expect(allReports[`${ONYXKEYS.COLLECTION.REPORT}1`]).toMatchObject(expectedReport);
+                        },
+                    });
+                }));
+
+        it('Should remove any instances of participants found in a report', () =>
+            Onyx.multiSet({
+                [`${ONYXKEYS.COLLECTION.REPORT}1`]: {
+                    reportID: 1,
+                    participants: ['fake@test.com'],
+                    participantAccountIDs: 5,
```

Comment:
```suggestion
                    participantAccountIDs: [5],
```
Have no significant impact just to comply with data types. (same change needed in expectedReport).

Reply:
Good catch.